perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/ExtSourcesManagerBlImpl.java

```diff
@@ -386,7 +386,7 @@ public Candidate getCandidate(PerunSession perunSession, Map<String,String> subj
 						try {
 							// Create new one if not exists
 							additionalExtSource = new ExtSource(additionalExtSourceName, additionalExtSourceType);
-							additionalExtSource = getPerunBl().getExtSourcesManagerBl().createExtSource(perunSession, additionalExtSource);
+							additionalExtSource = getPerunBl().getExtSourcesManagerBl().createExtSource(perunSession, additionalExtSource, attributes);
```

Comment:
These attributes are not extSource attributes. They are attributes of new candidate. There must be null.